src/main/java/com/github/studym/studymarathon/domain/board/entity/BaseEntity.java

```diff
@@ -0,0 +1,25 @@
+package com.github.studym.studymarathon.domain.board.entity;
+
+import lombok.Getter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.annotation.LastModifiedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import java.time.LocalDateTime;
+
+@MappedSuperclass
+@EntityListeners(value = {AuditingEntityListener.class})
+@Getter
+abstract class BaseEntity {
+    @CreatedDate
+    @Column(name = "created_at", updatable = false)
+    private LocalDateTime created_at;
+
+    @LastModifiedDate
+    @Column(name = "modified_at")
+    private LocalDateTime modified_at;
```

Comment:
마찬가지로 어차피 `@Column`으로 매핑이 되니 변수명은 camel case로 통일해주는 것이 좋을 것 같음

```suggestion
    @CreatedDate
    @Column(name = "created_at", updatable = false)
    private LocalDateTime createdAt;

    @LastModifiedDate
    @Column(name = "modified_at")
    private LocalDateTime modifiedAt;
```

Reply:
> 마찬가지로 어차피 `@Column`으로 매핑이 되니 변수명은 camel case로 통일해주는 것이 좋을 것 같음

camel case를 JPA로 쓰려니까 잘 손에 안익어서 그런거 같네

Reply:
근대 이거 같은경우엔 다시 보니까 그냥 원래대로 놔두는편이 더 좋을꺼 같기도 하네 흠